src/api/Visualization/types.ts

```diff
@@ -398,3 +398,26 @@ export type IUpdateWidgetParams = ICreateWidgetParams;
 export interface IDeleteWidgetParams {
   widgetKey: string;
 }
+
+export interface VisualizationCommentUser extends DeletableUser {
+  hasAccess: boolean;
+}
+
+export interface VisualizationComment {
+  id: number;
+  content: string;
+  user: VisualizationCommentUser;
+  createdAt: string;
+}
+
+export interface CreateVisualizationCommentParams {
+  visualizationId: number;
+  content: string;
+}
+
+export interface GetVisualizationCommentParams {
+  // Extends Datasource params
```

Comment:
I think it should extends IDataSourceParams since we are passing the sourceKey in the path.

WDYT @Diachk ?

Reply:
Yes indeed it should !